README.md

```diff
@@ -165,6 +165,23 @@ E.g. to apply a particular patch to a particular, known-good commit from the
                    --ref a870a02cc963de35452bbed932560ed69725c4f2 \
                    --pw https://patchwork.ozlabs.org/patch/886637
 
+To apply a local patch run:
+
+    $ skt --rc <SKTRC> --state --workdir <WORKDIR> -vv \
+             merge --baserepo <REPO_URL> \
+                   --ref <REPO_REF> \
+                   --patchlist <PATH_TO_PATCH>
+
+Here, `<REPO_REF>` would be the reference to checkout, and to apply the patch
+on top of, and `<PATH_TO_PATCH>` would be the patch file.
```

Comment:
Let's not repeat what `REPO_REF` is. Otherwise we would need to explain every placeholder every time, which would make the document hard to read.